src/PersonObjects/Sleeve/ui/TaskSelector.tsx

```diff
@@ -247,89 +240,49 @@ const canDo: {
   Synchronize: (sleeve: Sleeve) => sleeve.sync < 100,
 };
```

Comment:
Adding `as const` here will fix typing issues below, also you can remove the whole explicit type section.